elections/README.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Knative elections"
+linkTitle: "Knative elections"
+weight: 30
+type: "docs"
+---
+
+# Knative Elections
+
+This document outlines how to conduct a Knative Tecehnical Oversite Committee election. See [TOC election process](../mechanics/TOC.md) for more information of how the committee decides when to have elections, eligibility for voting, eligibility for candidacy, maximal representation, etc.
+
+## Process
+
+1. Steering committee prepare the election repository
+
+    * Make knative/community/elections/$YEAR
+    * Make knative/community/elections/$YEAR/README.md, this is the voter's guide
+        * Copy over the voter's guide from the previous year. The voter's guide is the single source of truth for the election that year! All annoucements and notices should link to this document.
+        * Update with new dates, candidates, and procedures (if necessary)
+    * Make knative/community/elections/$YEAR/voters.md
+        * Initial list is generated via the eligibility for voting requirements.
+        * Steering will update this list as voters submit the voting exemption form.
+
+1. Annouce voting schedule to the community
+
+    * Should be mostly links to the voter guide and the TOC charter.
+    * On knative-dev list, Knative Slack under #general, and twitter.
+
+3. Executing the Election in CIVS
+
+    * Use [CIVS](http://civs.cs.cornell.edu/civs_create.html) to create the election, which CIVS calls a poll. Once you send out the ballots you cannot UNSEND the emails, ensure everything in the form is correct!
+    * Name of the poll - Knative TOC Election for $YEAR”
+    * Name of supervisor - “Kubernetes Steering Committee”
+    * Email - election@knative.team
+    * Date and Time: Write in the date and time the election will stop. This field is not programmatic, the election is stopped by hand, so you can write this in plain text
+    * Description: Use the following text, modify it for either 3 or 4 positions, depending on the amount of open seats:
```

Comment:
```suggestion
    * Description: Use the following text, modify it for either 3 or 2 positions, depending on the amount of open seats:
```

I think we're going to have 5, alternating 3 and 2 seats?

Reply:
Right. Thanks for catching this.